Tests/New-PolarisRoute.Tests.ps1

```diff
@@ -23,6 +23,22 @@
         ( Get-PolarisRoute -Path $Path -Method $Method ).Scriptblock | Should Be $Path
     }
 
+
+        ## Fix for bug n°108 - Creating route with a lowercase 'get'
+
+        It "Should create route with lower case parameters" {
+            
+            New-PolarisRoute -Path /108 -Method get -ScriptBlock {
+                
+                $line = "<h1>this is H1</h1>"
+                $response.SetContentType("text/html");  
+                $response.send($line)
+            } -force
+            
+            (Get-PolarisRoute -Path 108).Method | should be 'GET'
+        }
+    
```

Comment:
nit pick, the indentation seems a bit off and there looks like an extra line at the top and bottom (should have 1 line empty between each test).

Can you just do a quick touch up of that? 😃

Reply:
sure!

I made the changes as requested. Let me know if something else is needed